### PR TITLE
(feat|COS-461): Created issues tab ui + connect with BE

### PR DESCRIPTION
**Pull request by Kasia9090**
What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

**Review by edifirut — DISMISSED** [review]

File: packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/IssuesPanel/IssueCard/IssueCard.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+  // const requestorName = getDisplayName(issue.requestedBy);
+  const statusColorScheme = (() => getStatusColor(issue.status))();
+
+  // const getLastCreatedNote = (notes: Array<Note>) => {
+  //   const sortedNotes = notes.sort((a, b) => b.createdAt - a.createdAt);
+  //   return sortedNotes[0].createdAt;
+  // };
+
+  console.log('🏷️ ----- issue.status: ', issue.status);
```
Comment (acalinica) [inline packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/IssuesPanel/IssueCard/IssueCard.tsx]: wild log.

File: packages/apps/spaces/app/src/hooks/useContactOrUserDisplayData/index.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    if (name.trim()) {
+      return name;
+    }
+
+    return contact?.emails?.[0]?.email
+      ? contact?.emails?.[0]?.email
+      : 'Unnamed';
+  }, []);
+};
```
Comment (acalinica) [inline packages/apps/spaces/app/src/hooks/useContactOrUserDisplayData/index.ts]: Just a suggestion, I've added during the migration [ts-pattern](https://github.com/gvergnaud/ts-pattern) which helped me write those kind of mappers with ease.

If you think it'll make sense to use it for mappers and parsers then you can have a look at the docs or at LastTouchpoint cell component where I've used it.

**Review by acalinica — DISMISSED** [review]

File: packages/apps/spaces/app/src/hooks/useContactOrUserDisplayData/index.ts (hunk 2 above)
Comment (Kasia9090) [reply packages/apps/spaces/app/src/hooks/useContactOrUserDisplayData/index.ts]: Thx, I like the approach, this hook is dead for now so I removed it but I'll have this in mind next time 👍

File: packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/IssuesPanel/IssueCard/IssueCard.tsx (hunk 1 above)
Comment (Kasia9090) [reply packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/IssuesPanel/IssueCard/IssueCard.tsx]: removed

**Review by acalinica — APPROVED** [review]

**Pull request merged**